test/Api.Test/Tools/Validators/SendRotationValidatorTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Bit.Api.Tools.Models.Request;
+using Bit.Api.Tools.Validators;
+using Bit.Core.Entities;
+using Bit.Core.Exceptions;
+using Bit.Core.Tools.Entities;
+using Bit.Core.Tools.Repositories;
+using Bit.Test.Common.AutoFixture;
+using Bit.Test.Common.AutoFixture.Attributes;
+using NSubstitute;
+using Xunit;
+
+namespace Bit.Api.Test.Tools.Validators;
+
+[SutProviderCustomize]
+public class SendRotationValidatorTests
+{
+    [Theory, BitAutoData]
```

Comment:
❓ Where do I go to find the test data this is running?

Reply:
The data is provided by AutoFixture through the attributes. Which means its auto-generated with some customization provided. AutoFixture has been pretty confusing for me to work with and I think we're going to move away from it in the future.
Most of the files for this is in `test/Common/AutoFixture`

Reply:
Without the data, there's no way to understand what is being tested. In a worst case scenario, the customizations end up replicating the computation under test.  Using a fixed data set is a crucial part of the test design when the system under test is deterministic. If the system is not deterministic or this is a specialized case--a fuzz test, for example--then automatically generated data is completely cool. 

⚠️ As written, I cannot understand what these tests are actually testing. We should probably huddle about how to address that instead of continuing here.

Reply:
Rewritten the tests to be deterministic. Please consider alerting the rest of the engineering team of these requirements for Tools code.